feat: Front UI/UX 기반 수정

chhs2131 opened the pull request:
## 📄 Summary
- 프론트 일부 UI를 모바일 환경에 맞게 개선합니다.
- 귀엽죠? 하하하

![image](https://github.com/user-attachments/assets/2f5a4534-fa96-4cea-b374-981e3524f4cc)


## 🙋🏻 More

![Aug-25-2024 00-14-35](https://github.com/user-attachments/assets/8a9e277e-d2f1-4b22-8e7d-8e44c5ba7ab1)


close #ui

yudonggeun committed:
fix: 포인트 충전 API 실패 오류 수정

yudonggeun committed:
chore: backend api url 변경 수정

chhs2131 committed:
Merge remote-tracking branch 'refs/remotes/origin/dev' into QA/ui

chhs2131 committed:
feat: 배민 폰트 적용

chhs2131 committed:
refactor: 포인트 충전 페이지 배경 흰색으로 변경

chhs2131 committed:
refactor: 버튼들 글자색을 하얀색으로 수정 (포인트충전, 로그인, 회원가입 버튼)

chhs2131 committed:
feat: 판매자 가입 기능 비활성화

chhs2131 committed:
feat: 2024-08-25 버전 (UI 개선)

HiiWee commented:
많이 귀여워요

HiiWee commented:
못하는게뭐람

HiiWee reviewed (dismissed):
쩌러여~ 😂

yudonggeun reviewed (dismissed):
폰트 수정, 하단 네비게이션 바 아이콘 수정, 거래 내역 UI 대폭! 개선

빨리 머지 해주세요! 현기증 나요.

chhs2131 committed:
feat: add CORS origins for localhost:3001 and front.dbsg.co.kr:3000

following origins:
  - http://localhost:3000
  - http://localhost:3001
  - http://front.dbsg.co.kr:3000

chhs2131 committed:
Merge remote-tracking branch 'refs/remotes/origin/dev' into QA/ui-2

# Conflicts:
#	src/main/resources/application.yaml

yudonggeun reviewed (approved):
LGTM
